core/parameters/src/parameter.rs

```diff
@@ -196,7 +196,7 @@ pub enum Parameter {
     // Contract runtime features
     #[strum(serialize = "disable_9393_fix")]
     Disable9393Fix,
-    FlatStorageReads,
+    _DeprecatedFlatStorageReads,
```

Comment:
revert this and rename `Parameter::_DeprecatedFlatStorageReads` to `Parameter::FlatStorageReads`